ecmascript/minifier/scripts/sort.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 set -eu
+
+# Fix sorting differences between Linux and Mac
+export LC_ALL=C
```

Comment:
This will unify the sort order. All test names are in ASCII, but when sorting using an UTF-8 locale, the `/` is ordered differently. Something about collations, which I really should know more about!

Reply:
Oh, I didn't know that.
Seems like a correct patch to me.